dev/VSIX/Extension/Cpp/WindowsAppSDK.Cpp.Extension.csproj

```diff
@@ -68,6 +69,9 @@
     <PackageReference Include="Microsoft.WindowsAppSDK" Version="[$(WindowsAppSDKVersion)]" GeneratePathProperty="true">
       <ExcludeAssets>All</ExcludeAssets>
     </PackageReference>
+    <PackageReference Include="Microsoft.Windows.SDK.BuildTools" Version="[$(WindowsSDKBuildToolsVersion)]" GeneratePathProperty="true">
+      <ExcludeAssets>All</ExcludeAssets>
+    </PackageReference>
```

Comment:
why do we need to add a reference explicitly? doesn't WindowsAppSDK depend on buildTools now?

Reply:
The WAS NuGet does depend on the BuildTools NuGet. When installing the WAS NuGet through NuGet Package Manager, the BuildTools NuGet is pulled in automatically. That's not true with the VSIX templates though. To quote Alan from an e-mail thread:

"The NuGet template wizard is used to install NuGet packages into a newly-created project and then restore them. It’s similar in spirit to Visual Studio’s NuGet Package Manager (which is what you used in your scenario), but doesn’t behave entirely the same. Specifically, when a project uses packages.config (all C++ projects), it *won’t* install package dependencies into the project. You can observe this behavior yourself by removing Microsoft.WindowsAppSDK.WinUI from one of our project templates, building a private VSIX, and creating a project… the package won’t be installed into the resulting .vcxproj even though it’s a dependency of Microsoft.WindowsAppSDK."

I found in my testing that the BuildTools package was not being installed with the C# templates too, so I addressed the C# case as well in this change.